test/functional/wallet_resendwallettransactions.py

```diff
@@ -39,6 +39,10 @@ def run_test(self):
         self.log.info("Create a new transaction and wait until it's broadcast")
         txid = int(node.sendtoaddress(node.getnewaddress(), 1), 16)
 
+        # Sleep for just over a second so the rebroadcast scheduler can get
+        # warmed up (see fNextResend in ResendWalletTransactions())
```

Comment:
style-nit in commit 9ad047d936c5601ca3841303595368e3c23e50b9:

```suggestion
        # warmed up (see nNextResend in ResendWalletTransactions(), which is called first scheduled 1sec after startup)
```

Reply:
fixed